File: SofiForce.Models/Models/ListModels/ItemStoreListModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ItemStoreListModel
    {

		public Int32? ItemStoreId { get; set; }
		public Int32? ItemId { get; set; }
		public Int32? BranchId { get; set; }
		public Int32? StoreId { get; set; }
		public Int32? Quantity { get; set; }
		public Int32? OnHand { get; set; }
		public DateTime? ExpireDate { get; set; }
		public string BatchNo { get; set; }
		public Int32? AcceptDays { get; set; }
		public string ItemCode { get; set; }
		public string ItemName { get; set; }
		public string VendorName { get; set; }
		public Int32? VendorId { get; set; }
		public Int32? Available { get; set; }

		public string StoreCode { get; set; }
		public string StoreName { get; set; }
		public string BranchCode { get; set; }
		public string BranchName { get; set; }

		public int? UnitId { get; set; }
		public bool? IsActive { get; set; }


		public int Quota { get; set; }


	}
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/ItemStoreListModel.ts
export class ItemStoreListModel {
    itemStoreId: number | null;
    itemId: number | null;
    branchId: number | null;
    storeId: number | null;
    quantity: number | null;
    onHand: number | null;
    expireDate: Date | null;
    batchNo: string;
    acceptDays: number | null;
    itemCode: string;
    itemName: string;
    vendorName: string;
    vendorId: number | null;
    available: number | null;
    unitId:number;
    isActive:boolean;
    
    storeCode: string;
    storeName: string;
    branchCode: string;
    branchName: string;

}